### DISCUSSION TITLE
guys help pls

### DISCUSSION THREAD

**Discussion by ArmolitskiyCarpoeb** [vgstation-coders/vgstation13#31411] [Q&A]
i dont have admin rank idk why 

i updated my repo and theres an old config so

(updated = no admin rank wtf)

i dont have admin rank idk why

**Comment by SECBATON-GRIFFON** [comment]
system 32 is a folder that prevents you getting it, the game detects it on your system so get rid of it

**Comment by Exxion** [comment]
I actually noticed this when joining from non-localhost a while ago. I think someone broke the legacy admin system.
